README.md

```diff
@@ -68,6 +68,19 @@ image = ...
 predictions = coco_demo.run_on_opencv_image(image)
 ```
 
+### Use it on an arbitrary GPU device
+For some cases, while multi-GPU devices are installed in a machine, a possible situation is that 
+we only have accesse to a specified GPU device (e.g. CUDA:1 or CUDA:2) for inference, testing or training.
+
+Here is an example for Mask R-CNN R-50 FPN quick on the second device (CUDA:1):
+```bash
+# for training
+python tools/train_net.py --config-file=configs/quick_schedules/e2e_mask_rcnn_R_50_FPN_quick.yaml MODEL.DEVICE cuda:1
```

Comment:
I would not recommend this as the first method for running on the a different GPU.

Indeed, this used to still allocate some memory on the first GPU. Nowadays this is still the case but much less (~10MB from what I tried), while using `CUDA_VISIBLE_DEVICES` really doesn't use anything.

After you make this change and add first a note on this env var then I think this PR is good to go